.circleci/config.yml

```diff
@@ -7,14 +7,21 @@ orbs:
 
 executors:
   docker-python:
-      docker:
-        - image: circleci/python:3.7
+    docker:
+      - image: circleci/python:3.7
   docker-terraform:
-     docker:
-       -  image: "hashicorp/terraform:light"
+    docker:
+      - image: "hashicorp/terraform:light"
```

Comment:
My VSCode clearly has a lot of opinions about yml formatting - let me know if problematic and I will restore the original formatting.